common/containers/Tabs/Swap/components/LiteSend/Fields.tsx

```diff
@@ -31,7 +31,7 @@ class FieldsClass extends Component<Props> {
               onClick={this.changeWallet}
             >
               <i className="fa fa-refresh" />
-              {translate('Change Wallet')}
+              {translateRaw('Change Wallet')}
```

Comment:
Should be a key and not raw text. Might also consider making the change wallet button a component if that makes any sense, I see it a whole lot.